.github/workflows/tests-e2e.yml

```diff
@@ -130,8 +130,8 @@ jobs:
         browser: ['chrome']
         wp: ['5.7']
         snapshots: [false]
-        # We want to split up the tests into 2 parts running in parallel.
-        shard: ['1/2', '2/2']
+        # We want to split up the tests into 4 parts running in parallel.
+        shard: ['1/4', '2/4', '3/4', '4/4']
```

Comment:
Splits the WP 5.7 run into 4 shards --- this helps as it splits the number of test per shard up and in turn when re-running failed tests the number of test will be lower.   Also should speed up the 5.7 tests.